packages/core/src/TimeField/TimeFieldRoot.vue

```diff
@@ -103,6 +120,12 @@ const props = withDefaults(defineProps<TimeFieldRootProps>(), {
   readonly: false,
   placeholder: undefined,
   isDateUnavailable: undefined,
+  yearStep: 1,
+  monthStep: 1,
+  dayStep: 1,
+  hourStep: 1,
+  minuteStep: 1,
+  secondStep: 1,
```

Comment:
I think adding each step for each field is abit excessive. What do you think if we have a `step` prop that contain different key for different field. we can use [defu](https://github.com/unjs/defu) to populate default value for field not provided in the prop.
 
eg:
```ts
type Step = {
  day?: number
  month?: number
  ...
}

const props = defineProps<Props & { step?: Step }>()

const step = computed(() => {
  return defu(props.step, {
    day: 1,
    month: 1
    ...
  })
})
```

Reply:
Nice idea 👍.
Now updated to an aggregated prop `step`